### PR TITLE
Neural Net

### PR DESCRIPTION
**Pull request by chingyap1**

### CODE REVIEW

**Comment by doctortheisen** [comment]
@chingyap1 Is this ready for merge? The pull request status is Draft

**Comment by doctortheisen** [comment]
I'm working on the neural network right now, to incorporate preprocessing:
- normalizing numerics
- handling cats

Will post shortly.

**Comment by chingyap1** [comment]
> I'm working on the neural network right now, to incorporate preprocessing:
> 
> * normalizing numerics
> * handling cats
> 
> Will post shortly.

Maybe we can merge our work here, I think my numeric normalization looks good so hopefully we had similar results. I'm currently dropping binary categoricals, would be interesting if you found a nice way to include them without destroying gradients.

File: main.Rmd [hunk 1]
```diff
@@ -389,13 +389,61 @@ r2_tree = 1-R2(test_data$price,predict(prost_tree, test_data))
...
+## Triple node neural net
+```{r, eval=FALSE}
+set.seed (1023)
+# Dropping binary factors from OLS best
+nn2_model <- neuralnet(price ~ bedrooms + bathrooms + sqft_living + view + sqft_living15 + meanPrice_for_zipcode + yr_built + days_since_first + floors + condition + sqft_lot15 + sqft_lot, data = train_norm, threshold = 0.05, rep = 2, hidden = 3)
+plot(nn2_model)
+
+nn2_results <- compute(nn2_model, test_norm)
+cor(nn2_results$net.result, test_norm$price)^2
+```
```
Comment (chingyap1) [inline main.Rmd:434]: Should we delete the first two tries and just show final result?

Comment (doctortheisen) [reply main.Rmd:434]: Very nice. I agree with you from what I can see so far. Will test a bit more and report back.

Comment (chingyap1) [reply main.Rmd]: Any update? I'll need to merge this and align with the email version sometime tonight...

Comment (doctortheisen) [reply main.Rmd]: @chingyap1 apologies. got stuck at work and I'm going through code now. The nn_model I was developing took an hour per render and threw things off.

**Pull request merged**